<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionGrantSkills</class>
 <widget class="QWidget" name="ActionGrantSkills">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Source Context:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="sourceContext">
       <item>
        <property name="text">
         <string>Source</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Party</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="location">
       <item>
        <property name="text">
         <string>World</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Channel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Instance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zone</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Skill Points:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="skillPoints">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Expertise Points:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ActionMap" name="expertisePoints" native="true"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Skill IDs:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ActionIDs" name="skillIDs" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Target Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="targetType">
       <item>
        <property name="text">
         <string>Character</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Partner</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionMap</class>
   <extends>QWidget</extends>
   <header>ActionMap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ActionIDs</class>
   <extends>QWidget</extends>
   <header>ActionIDs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
